microshift_troubleshooting/microshift-troubleshoot-backup-restore.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: ASSEMBLY
+[id="microshift-troubleshoot-backup-restore.adoc"]
+= Troubleshoot data backup and restore
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-troubleshoot-data-backup-and-restore
+
+toc::[]
+
+To troubleshoot failed data backups and restorations, check the basics first, such as data paths, storage configuration, and storage capacity.
+
+[id="troubleshoot-backup-restore-microshift-backup-data-failed"]
+== Backup data failed
+Data backups are automatic on `rpm-ostree` systems. If you are not using an `rpm-ostree` system and attempted to create a manual backup, the following reasons could cause the backup to fail:
+
+* If {product-title} stopped running because of an error, you cannot perform a backup of the data.
+** First, make sure the system is healthy.
+** Then, stop it in a healthy state before attempting a backup.
+** You might need to wait several minutes after a system start to successfully stop {product-title}. Allow the system to complete health checks and any other background processes before attempting to back up.
```

Comment:
Line 17 says to stop the system, but line 18 implies that you started the system. I'm not sure what to suggest because I'm not sure what the actual procedure entails, but I think this needs some clarification.